command/agent/exec/exec.go

```diff
@@ -0,0 +1,260 @@
+package exec
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"time"
+
+	"github.com/hashicorp/consul-template/child"
+	ctconfig "github.com/hashicorp/consul-template/config"
+	"github.com/hashicorp/consul-template/manager"
+	"github.com/hashicorp/go-hclog"
+
```

Comment:
Nit: delete this empty line

Reply:
that's intentional by `gofmt`, imports are grouped
stdlib
external/third party
internal/first party

Reply:
Oh, interesting. I'm not sure if it's different with `gofumpt`, but I've made an effort to delete these when they pop up in my PRs, and it seems to auto-organize them when I do. I'm happy to keep this as-is!

Reply:
just ran `make fmt` and no change? maybe we need to add some import formatting 🤷